package.json

```diff
@@ -40,21 +40,18 @@
     "prepublishOnly": "npm run compile",
     "npub:precheck": "npm run lint && npm run format:check && npm run ts:check && npm run test"
   },
-  "dependencies": {
-    "tailwindcss": "^2.0.0"
-  },
   "peerDependencies": {
```

Comment:
yeah, you're right, i didn't realize that npm removed this. I think `>=2` is probably correct.  or should it just be `^3.0.0`?

Reply:
Imho, it could be >= 2, because the lib is compatible with 2 and newer ones.

Reply:
Personally, I wouldn't include tailwind as dependency, just as peerDependency like you're doing with `react-native`.

If you add tailwindcss as dependency, will be multiples versions of tailwind installed if you have other projects (for example a monorepo), or for example locally installed and then inside this library another tailwind. 

Making it peerDependency your users, can install any tailwind version that they're comfortable with.

Reply:
regarding a peerDependency, I agree with you in theory. in fact, that's how the lib was setup at the beginning.  but i got _so many_ issues from people who ignored the peer dep warning, and just pasted in some big node error about not being able to resolve `tailwindcss`.  Once I added it as a true dependency, that type of issue went away, and I don't think it hurts anyone, since the range is so lax.  so, really it's a quality of life issue for me.

Reply:
Alright, sounds reasonable enough =) 
Feel free to ping me if you need help with something. I'm using your library to build from scratch a company and it's such a pleasure :) 

If someday we have some revenue we'll destinate the first income to sponsor your project :)